lib/cadet/assessments/submission_votes.ex

```diff
@@ -17,6 +17,7 @@ defmodule Cadet.Assessments.SubmissionVotes do
   @required_fields ~w(voter_id submission_id question_id)a
   @optional_fields ~w(score)a
 
+  # |> unique_constraint(:unique, name: :unique_score) (Unique constraint removed)
```

Comment:
For here, change the comment to a factual statement like "there is no unique constraint...blah blah".